src/plugins/intel_cpu/src/nodes/subgraph.cpp

```diff
@@ -693,12 +698,12 @@ bool Snippet::SnippetJitExecutor::optimizeExecDomain(std::vector<VectorDims>& in
 
 void Snippet::SnippetJitExecutor::generate(const jit_snippets_compile_args* jcp) {
     using Manager = snippets::pass::Manager;
+    std::vector<Manager::PositionedPass> backend_passes;
+#if defined(OPENVINO_ARCH_X86_64)
```

Comment:
```suggestion
#if defined(OPENVINO_ARCH_X86_64)
#define SNIPPETS_REGISTER_PASS(PASS_POS, PASS, ...) \
        backend_passes.emplace_back(PASS_POS, std::make_shared<PASS>(__VA_ARGS__))
#else
#define SNIPPETS_REGISTER_PASS(PASS_POS, PASS, ...)
#endif // OPENVINO_ARCH_X86_64

SNIPPETS_REGISTER_PASS(PassPosition(Place::PipelineStart), ConvertToSwishCPU);
....
#undef SNIPPETS_REGISTER_PASS
```

Probably, it's a matter of taste. But for me it looks better.

Reply:
updated